src/Raku/ast/expressions.rakumod

```diff
@@ -3,7 +3,10 @@ class RakuAST::CaptureSource
   is RakuAST::Node { }
 
 # Everything that can appear as an expression does RakuAST::Expression.
+# Furthermore, since every expression can exist as statement, consider
```

Comment:
That doesn't sound right. While every expression can exist as a statement, those are represented in the AST as RakuAST::Statement::Expression, i.e. the RakuAST::Expression is wrapped in a statement. I think that's an important distinction that we want to retain.

Reply:
Hmm, then we need to be more careful about what ends up in a (supposed-to-be) `RakuAST::Blorst` attribute. But you are probably right, instead the issue is that something is passing a `RakuAST::Expression` instead of a `RakuAST::Statement::Expression`.

If you don't mind, I'll have a look at this in a separate PR since I don't think this alone should result in a revert.

Reply:
Looks like the issue was with the original test, actually. I'll fix this up.